### PR TITLE
Create HWindex.html

### PR DESCRIPTION
**Pull request by AnnaLytvyn**
please check my fomework

### CODE REVIEW

File: HWindex.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<!--suppress ALL -->
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My homework4</title>
+</head>
+<body>
+<head>
```
Comment (marynabudz1) [inline HWindex.html:11]: should be header
head is only one right above body

File: HWindex.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <h1>Form</h1>
+</head>
+<form action="">
+    <label for="name">First name</label>
+    <input type="text" id="form" name="form"><br><br>
```
Comment (marynabudz1) [inline HWindex.html:16]: id should be name not form

File: HWindex.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    <label for="email">Email</label>
+    <input type="email" id="email" name="email"><br><br>
+    <label for="lname">Last name</label>
```
Comment (marynabudz1) [inline HWindex.html:19]: please provide more relevant name 
lname - is not understandable 
last-name - better naming

File: HWindex.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    <input type="text" id="lname" name="name"><br><br>
+    <label for="password">Password</label>
+    <input type="password" id="password" name="password"><br><br>
+    <label for="password">Password again</label>
+    <input type="password" id="password" name="password"><br><br>
```
Comment (marynabudz1) [inline HWindex.html:24]: id shoud be unique value

File: HWindex.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    <label for="gender">Gender</label>
+    <input type="checkbox" id="female" name="female" checked>
+    <label for="checkbox" id="female">female</label>
```
Comment (marynabudz1) [inline HWindex.html:27]: label should not have id

File: HWindex.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    <input type="checkbox" id="female" name="female">
+    <label for="female" id="female">female</label><br><br>
```
Comment (marynabudz1) [inline HWindex.html:29]: label should not have id

File: HWindex.html [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+        <option value="projectMmnagement">Project Management</option>
+        <option value="qamanual">QA Manual</option>
+    </select>
+    <br><br>
+    <label for="adress">Adress</label>
+    <textarea name="adress" id="adress" cols="30" rows="10"
+              placeholder="country, region, city, town, street"></textarea>
+    </select><br>
+    <br>
+    <input type="checkbox">
```
Comment (marynabudz1) [inline HWindex.html:43]: missing id for <input type="checkbox">

File: HWindex.html [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+    <label for="checkbox">I agree with my personal data</label>
+    <br><br>
+    <input type="reset" value="Reset">
```
Comment (marynabudz1) [inline HWindex.html:46]: better use buttons

**Review by marynabudz1 — APPROVED** [review]

File: HWindex.html (hunk 6 above)
Comment (AnnaLytvyn) [reply HWindex.html:29]: thank you, noted

**Pull request merged**